src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-chart/cephfs-chart.component.spec.ts

```diff
@@ -9,6 +9,12 @@ describe('CephfsChartComponent', () => {
   let component: CephfsChartComponent;
   let fixture: ComponentFixture<CephfsChartComponent>;
 
+  const createArray = () => {
```

Comment:
`createArray` always returns the same result. It does not need to be a function.

Reply:
So would it make more sense to create this array statically instead?

Reply:
> So would it make more sense to create this array statically instead?

I'd say so. Currently, the function is called two times, returning the same result. That's not much, but image we'd do it all the time. I'd prefer to have it as an array and pass that directly to reduce clutter. If there's a need to return variations of data, the function could easily be reimplemented and would then serve an IMO meaningful purpose.